tools/genform/genform.xml

```diff
@@ -0,0 +1,495 @@
+                <has_text text=" oei " />
+                <has_text text=" dbeexc=1" />
+                <not_has_text text=" ivsm2mv=" />
+                <not_has_text text=" ivsm2ap2=" />
+                <not_has_text text=" oms=" />
+                <not_has_text text=" omsms=" />
+                <not_has_text text=" oclean=" />
+                <not_has_text text=" analyze " />
+                <not_has_text text=" loss " />
+                <not_has_text text=" intens " />
+                <not_has_text text=" dbe " />
+                <not_has_text text=" cm " />
+                <not_has_text text=" pc " />
+                <not_has_text text=" sc " />
+            </assert_command>
+        </test>
+    </tests>
+    <help><![CDATA[
+        
+**Overview**
+
+Genform generates candidate molecular formulas from high-resolution
+MS data. It calculates match values (MV) that show how well candidate molecular formulas fit the MS
+isotope peak distributions (MS MV) and the high-resolution MS/MS fragment peak masses (MSMS
+MV). Finally it computes a combined match value from these two scores. This software can be
+regarded as a further development of the ElCoCo and MolForm modules of MOLGEN-MS with a clear
+specialization towards MS/MS.
+
+**Input**        
+
+MS and MS/MS centroided spectra are given as tab separated files with two columns:
+
+1. m/z
+2. intensity
+
+**Output**
+
+Main output containing a list of all possible molecular formulas, further options can be chosen:
+
+1. if the explained peaks shall also be written to the output
+2. if the explanation shall be given as molecular formulas for the fragment or as neutral losses
+3. double bond equivalents for each molecular formula
+4. calculated ion masses 
+5. output match values in percent
+
+Output structure:
+
+molecular fomula / (double bond equivalent) / (calculated mass) / ppm / MS MV / MSMS MV / combined MV 
+
+ppm: The value ppm denotes the relative deviation of
+the calculated mass of the molecular ion from the mass of the basepeak (basemass) in the MS in
+parts per million (ppm).
+
+MS MV: It is calculated from measured and
+theoretical isotope peak intensities. The MV ranges from 0 to 1. The better theoretical and measured
+isotope distributions fit, the higher is the MV. There are three different algorithms implemented for
+calculating the MS MV, which are nomalized dot product, nomalized sum of squared errors and the normalized sum of absolute errors.
+
+MSMS MV: It shows how well the masses of the MS/MS fit possible fragments of the candidate molecular formula. Again, the MS/MS MV is a value
+between 0 and 1, whereby higher values reflect a better fit. 
+
+combined MV: combined match value
+
+Example output:
+
+======== === =========== ====== ======== ========= ========
+C4H9N3O2  2   132.07675   -0.8   96.674   100.000   96.674
+======== === =========== ====== ======== ========= ========
+
+**Advanced options**
+
+...
```

Comment:
Can be frustrating :)

Reply:
I know, will change that...
is a link to the command line tool manual sufficient enough?
[https://www.researchgate.net/publication/307964728_MOLGEN-MSMS_Software_User_Manual](url)

Reply:
It's ok for me

Reply:
it's ok for me too. Yes the information available in the yaml file is enough but it is not evident for anybody to find the information in this file (more in a README ou markdown file) ;-)

Reply:
Do you know the HR2 algorithm developped by Tobias Kind and Oliver Fiehn. (2007) ? Publication is I think "Seven Golden Rules for heuristic filtering of molecular formulas obtained by accurate mass spectrometry." BMC Bioinformatics p8:105 http://www.ncbi.nlm.nih.gov/pubmed/17389044
HR2 original program: http://fiehnlab.ucdavis.edu/projects/Seven_Golden_Rules/Software

Reply:
Ok so I add a link to the commandline manual